### PR TITLE
LPS-110734 Remove Simple WYSIWYG Support

### PR DESCRIPTION
**Pull request by julien**

### CODE REVIEW

**Comment by julien** [comment]
ci:test:sf

**Comment by julien** [comment]
ci:test:stable

**Comment by julien** [comment]
ci:test:relevant

**Comment by julien** [comment]
Some notes about these changes.

After doing an `ant all` on this branch, the simple editor module is not active.

```sh
~$ telnet localhost 11311
Trying 127.0.0.1...
Connected to localhost.
Escape character is '^]'.
____________________________
Welcome to Apache Felix Gogo

g! lb simple
No matching bundles found
g! lb editor
START LEVEL 20
   ID|State      |Level|Name
  396|Active     |   15|Liferay Frontend Image Editor Integration Document Library (4.0.7)|4.0.7
  446|Active     |   10|Liferay Journal Editor Configuration (4.0.2)|4.0.2
  472|Active     |   10|Liferay Blogs Editor Configuration (4.0.3)|4.0.3
  552|Active     |   10|Liferay Comment Editor Configuration (4.0.2)|4.0.2
  584|Active     |   10|Liferay Adaptive Media Journal Editor Configuration (4.0.2)|4.0.2
  586|Active     |   10|Liferay Frontend Editor Lang (5.0.4)|5.0.4
  589|Active     |   15|Liferay Frontend Image Editor Web (4.0.12)|4.0.12
  610|Active     |   10|Liferay Wiki Editor Configuration (4.0.2)|4.0.2
  613|Active     |   15|Liferay Frontend Editor TinyMCE Web (4.0.3)|4.0.3
  625|Active     |   10|Liferay Item Selector Editor Configuration (5.0.2)|5.0.2
  699|Active     |   15|Liferay Frontend Editor AlloyEditor Web (4.0.6)|4.0.6
  724|Active     |   10|Liferay Announcements Editor Configuration (3.0.2)|3.0.2
  753|Active     |   10|Liferay Message Boards Editor Configuration (4.0.2)|4.0.2
  759|Active     |   10|Liferay Knowledge Base Editor Configuration (4.0.3)|4.0.3
  769|Active     |   15|Liferay Frontend Editor CKEditor Web (4.0.6)|4.0.6
  775|Active     |   10|Liferay Layout Content Page Editor API (2.1.2)|2.1.2
  778|Active     |   10|Liferay Frontend Editor API (5.0.2)|5.0.2
  801|Active     |   15|Liferay Layout Content Page Editor Web (2.0.29)|2.0.29
  863|Active     |   10|Liferay Frontend Image Editor API (4.0.3)|4.0.3
  872|Active     |   15|Liferay Frontend Editor Taglib (3.0.3)|3.0.3
  940|Resolved   |   10|Liferay Reading Time Editor Plugin (3.0.3)|3.0.3
  942|Active     |   15|Liferay Wiki Engine Input Editor Common (6.0.4)|6.0.4
  948|Active     |   10|Liferay Wiki Editor Link Browse Web (3.0.2)|3.0.2
  951|Active     |   10|Liferay Adaptive Media Blogs Editor Configuration (4.0.2)|4.0.2
g! 
```

If we navigate to `./modules/apps/archived/frontend-editor-simple-web` and execute `gradlew deploy` we see the following message in tomcat's log:

```sh
2020-03-30 10:04:13.464 INFO  [fileinstall-/home/jc/Documents/portal/bundles/osgi/modules][BundleStartStopLogger:46] STARTED com.liferay.frontend.editor.simple.web_4.0.3 [962]
```


and we if connect to the gogo shell we can also confirm that this module is active 

```sh
~$ telnet localhost 11311
Trying 127.0.0.1...
Connected to localhost.
Escape character is '^]'.
____________________________
Welcome to Apache Felix Gogo

g! lb simple
START LEVEL 20
   ID|State      |Level|Name
  962|Active     |   15|Liferay Frontend Editor Simple Web (4.0.3)|4.0.3
g!
```

if we add these properties to the `portal-ext.properties` file 

```properties
jeditor.wysiwyg.default=simple
editor.wysiwyg.portal-web.docroot.html.portlet.blogs.edit_entry.jsp=simple
editor.wysiwyg.portal-web.docroot.html.portlet.journal.edit_article_content.jsp=simple
editor.wysiwyg.portal-impl.portlet.ddm.text_html.ftl=simple
```

and restart tomcat, when navigating to the blog creation page we can see that simple editor is being used

blogs_simple_editor

File: readme/BREAKING_CHANGES.markdown [hunk 1]
```diff
@@ -499,4 +499,30 @@ Remove those properties from your properties file.
 
 This feature has been deprecated.
 
----------------------------------------
\ No newline at end of file
+---------------------------------------
+
+### Simple Editor is no longer bundled by default
```
Comment (jbalsas) [inline readme/BREAKING_CHANGES.markdown]: @julien, @mateomustapic, see similar feedback at https://github.com/julien/liferay-portal/pull/256/files

**Comment by julien** [comment]
Sent [here](https://github.com/jbalsas/liferay-portal/pull/2156)

**Pull request closed without merging**